src/main/groovy/org/mockito/release/internal/gradle/BaseJavaLibraryPlugin.java

```diff
@@ -90,5 +91,9 @@ public void execute(MavenPublication publication) {
 
         //so that we flesh out problems with maven publication during the build process
         project.getTasks().getByName("build").dependsOn("publishToMavenLocal");
```

Comment:
@szczepiq It is unrelated. I was just curios seeing that code. Once, I will come back to CD in FOSS there is a chance for contribution ;).